setup.py

```diff
@@ -65,7 +65,7 @@ def get_version(version_tuple):
       long_description=LONG_DESCRIPTION,
       platforms=['any'],
       classifiers=CLASSIFIERS,
-      install_requires=['pymongo>=3.0,<3.14'],
+      install_requires=['pymongo==4.2.0'],
```

Comment:
This change is surprising. Unlike `requirements.txt` (which is used mostly for local development and for CI), it's fine to be more lax in `install_requires`. Typically it's enough to just ensure that you're sticking to one `major` version, e.g. `>=4.2,<5.0`.